XmlValidator/spec_common.xml

```diff
@@ -1508,7 +1512,11 @@ for key in text:
         text[key] = text[key][1:]
     else:
         minus = ''
-    text[key] = minus + r'\text{' + utils.format_number(text[key], thousandsSeparator=r'\ ') + '}'
+    if '|' in text[key]:
+        index = text[key].find('|')
+        text[key] = minus + r'\text{' + utils.format_number(text[key][:index]) + ' | ' + text[key][index+1:] + '}'
```

Comment:
Going to handle this one from validator-updates-round-1b branch as that has more up to date code for the number tag: https://github.com/Siyavula/cnxml-validator/pull/15/files. Apologies, this whole project has ended up being too messy